Civi/Token/TokenCompatSubscriber.php

```diff
@@ -148,8 +147,7 @@ public function onRender(TokenRenderEvent $e) {
     $e->string = \CRM_Utils_Token::replaceDomainTokens($e->string, $domain, $isHtml, $e->message['tokens'], $useSmarty);
 
     if (!empty($e->context['contact'])) {
-      \CRM_Utils_Token::replaceGreetingTokens($e->string, $e->context['contact'], $e->context['contact']['contact_id'], NULL, $useSmarty);
```

Comment:
I'm starting to thing that the only reason this is required is to do the same thing the next row does - but with `$replaceBlankToken` set to `NULL|FALSE`